.github/workflows/CI.yml

```diff
@@ -22,3 +22,30 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         continue-on-error: true
+
+  build:
+    name: build and upload artifacts
+    strategy:
+      matrix:
```

Comment:
```suggestion
      target:
```

Reply:
`matrix` is a keyword that (AFAIK) cannot be changed. ([Strategy Doc](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix), [Matrix Doc](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs))
I can try to make that change, and revert the change if it does not work, but I really don't think it will work.